ansible/adhoc/backup-keytabs.yml

```diff
@@ -0,0 +1,11 @@
+# Use ONE of the following tags on this playbook:
```

Comment:
Could this be made an extravar instead e.g. `backup_keytabs_action=deploy`? I think it would make it easier when we come to use this with Azimuth.